qanta/wikipedia/cached_wikipedia.py

```diff
@@ -145,6 +145,9 @@ def __getitem__(self, key: str):
             except ImportError:
                 print("Error importing %s" % key)
                 page = None
+            except ValueError:
+                print("Error importing %s" % key)
+                page = None
```

Comment:
An undetected ValueError was causing the program to Abort.  I only expanded the list of handled errors but a better way would be to handle all undefined errors.